docs/src/database.rst

```diff
@@ -35,14 +35,45 @@ way described in your distribution's web pages.
    * `Mongo Shell Quick Reference <https://docs.mongodb.com/manual/reference/mongo-shell/>`_
    * `Tutorialspoint <https://www.tutorialspoint.com/mongodb/mongodb_create_database.htm>`_
    * `ArchLinux wiki <https://wiki.archlinux.org/index.php/MongoDB>`_
+Setup MongoDB without root access
+-------------------------------
 
-Create a MongoDB
-----------------
+As mentioned in  `Mongo docs <https://docs.mongodb.com/manual/tutorial/install-mongodb-on-debian/#using-tgz-tarballs>`_ download MongoDB, extract it and make sure the binaries are in a directory listed in your PATH environment variable. Next create the database using::
+
+      mongo orion_test --eval 'db.createUser({user:"user",pwd:"pass",roles:["readWrite"]});'
+   
+To start MongoDb::
 
-Invoke the following command as you can read `here <https://docs.mongodb.com/manual/reference/method/db.createUser/>`_::
+      mongod --dbpath /path/to/database
+
+Setup MongoDB with root access
+----------------
+Follow the instructions described in  `Mongo docs <https://docs.mongodb.com/manual/administration/install-on-linux/>`_. If you have root access you can invoke the following command as you can read `here <https://docs.mongodb.com/manual/reference/method/db.createUser/>`_::
 
    mongo orion_test --eval 'db.createUser({user:"user",pwd:"pass",roles:["readWrite"]});'
 
+And start MongoDB::
+
+   sudo service mongod start      
+
+Atlas MongoDB
+=============
+1. Create an account `here <https://www.mongodb.com/cloud/atlas>`_.
+2. Follow the defaults to create a free cluster.
+3. Add cluster name and click on "Create Cluster".
+4. Wait for the cluster to be created.
+5. In "Overview" tab, click on "CONNECT".
+6. Add the IP of your compuer to the whitelist or "Allow access from anywhere."
```

Comment:
It's in "Security tab", then "IP Whitelist" subtab. This should be 5., not 6.

Reply:
It also exists in the overview tab. I thought it would be easier this way.

Reply:
It's your call though.

Reply:
Oh, that's true!

Reply:
It's good this way. Better be simple. :+1: